#!/usr/bin/env python                                                                                                      

PACKAGE="detection_hg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("hm_thresh", double_t,  0,  "minimum hm peak value", 0, 0, 1)
gen.add("vis_kp_ind", int_t,  0,  "vis_kp_ind", -1, -1, 19)
gen.add("circle_rad", int_t,  0,  "circle rad", 1, 1, 5)
gen.add("font_thick", int_t,  0,  "thickness of font line", 3, 1, 10)
gen.add("font_scale", double_t,  0,  "scale of font", 0.2, 0, 2)


exit(gen.generate(PACKAGE, "detection_hg", "param"))
